lib/networkCall.js

```diff
@@ -0,0 +1,20 @@
+const axios = require('axios')
+const logger = require('../utils/logger')
+
+const networkCall = async (url, method = 'get', queryParams = null, body = null, headers = null) => {
```

Comment:
Can we also add `options = {}` argument to the function and add `...options` in the axios call.
This will be useful in case we need to add other provided oprions to the call rather than the frequently used ones.

Add JSDOC.

And, `networkCall` seems a long name for the function which is to be used too frequently in the codebase.
Can we change the name to anything like `request` or anything else short? 😬